content/references/xrp-ledger-toml.md

```diff
@@ -0,0 +1,308 @@
+peer = "https://s1.ripple.com:51235/"
+desc = "General purpose server cluster"
+
+[[SERVERS]]
+json_rpc = "https://s2.ripple.com:51234/"
+ws = "wss://s2.ripple.com/"
+peer = "https://s2.ripple.com:51235/"
+desc = "Full-history server cluster"
+
+[[SERVERS]]
+json_rpc = "https://s.testnet.rippletest.net:51234/"
+ws = "wss://s.testnet.rippletest.net:51233/"
+peer = "https://s.testnet.rippletest.net:51235/"
+network = "testnet"
+desc = "Test Net public server cluster"
+
+[[PRINCIPALS]]
+name = "Rome Reginelli" # Primary spec author
+email = "rome@example.com" # Not my real email address
+
+[[CURRENCIES]]
+code = "LOL"
+issuer = "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn"
+network = "testnet"
+display_decimals = 2
+symbol = "😆" # In practical situations, it may be unwise to use emoji
+
+# End of file
+```
+
+### Metadata
+
+The metadata section ( `[METADATA]`) provides information about the `xrp-ledger.toml` file itself. You MAY provide any of the following fields (case-sensitive):
+
+| Field      | Type             | Description                                  |
+|:-----------|:-----------------|:---------------------------------------------|
+| `modified` | Offset Date-Time | The time the `xrp-leder.toml` file was last modified.    |
+| `expires`  | Offset Date-Time | If the current time is equal or greater than this time, the `xrp-ledger.toml` file should be considered expired. |
+
+The specification does not define a `domain` field; the field should be determined from the site serving the file.
+
+**Tip:** For Offset Date-Time values, Ripple RECOMMENDS that you use the offset `Z` and provide precision up to milliseconds. (For example, `2019-01-22T22:26:58.027Z`) If you edit the file by hand, you MAY approximate the time by providing zeroes for the hours, minutes, seconds, and milliseconds. (For example, `2019-01-22T00:00:00.000Z`)
+
+### Validators
+
+The validators list (`[[VALIDATORS]]`) provides information about validating servers you operate. Each entry describes a separate validating server.
+
+The _first_ `[[VALIDATORS]]` entry in the file is treated as your primary validator. If you operate one or more validators for the production XRP Ledger, you should put the one you want others to trust first.
+
+For _each_ `[[VALIDATORS]]` entry, you MAY provide any of the following fields:
+
+| Field        | Type   | Description                                          |
+|:-------------|:-------|:-----------------------------------------------------|
+| `public_key` | String | The master public key of your primary validator, encoded in the XRP Ledger's base58 format (typically, this starts with `n`). |
+| `network`  | String | Which network chain this validator follows. If omitted, clients SHOULD assume that the validator follows the production XRP Ledger. Use `main` to explicitly specify the production XRP Ledger. Use `testnet` for Ripple's XRP Ledger Test Net. You MAY provide other values to describe other test nets or non-standard network chains. |
```

Comment:
I expect the Data API to change from using `altnet` to `testnet` in API requests/responses in a forthcoming version, for consistency.

Reply:
Fyi, there are other mentions of "altnet" in this repo, might be worth a grep.